test/include/seqan3/test/tmp_directory.hpp

```diff
@@ -157,18 +157,6 @@ class tmp_directory
         return exists(directory_path.value()) && is_empty(directory_path.value());
     }
 
-    /*!\brief Removes all files from the temporary directory
-     */
-    void clean()
-    {
-        assert(directory_path);
-
-        // Deletes all directories recursivly without following symlinks.
-        // Function will throw on error.
-        std::filesystem::remove_all(directory_path.value());
-        directory_path = std::nullopt;
-    }
-
 private:
     /*!\brief Warns and cleans if directory is not empty
      */
```

Comment:
`clean()` would be a better name since no more warnings are emitted? Also adapt documentation.